index.bs

```diff
@@ -1051,7 +1018,7 @@ When the <dfn method for="WebGLRenderingContextBase">setCompatibleXRDevice(|devi
 
 Additionally, when any WebGL [=handle the context loss|context is lost=] run the following steps prior to firing the "webglcontextlost" event:
 
-  1. Set the context's [=compatible XR device=] to <code>null</code>.
+  1. Set the context's [=XR compatible=] boolean to <code>false</code>.
```

Comment:
We need to explicitly call out the behavior of the compatible webgl context when a device change happens.

Reply:
Can you point me to the line number?  I'm not seeing where we explain that the XR compatible boolean gets reset to false when the device change happens.

Reply:
Ah, you meant you *just* added it. ^_^